src/State/GameMenu.cpp

```diff
@@ -82,37 +83,36 @@ void GameMenu::init()
     // label: save game
     auto saveGameButtonLabel = new UI::TextArea(_t(MSG_OPTIONS, 0), backgroundX+8, backgroundY+26);
     saveGameButtonLabel->setFont(font);
-    saveGameButtonLabel->setWidth(150);
+    saveGameButtonLabel->setSize({150, 0});
```

Comment:
Then we need to refactor A LOT of code, we have those magic number all over the place. Since we plan to re-write all states in Lua, I think we can start using better programming practices there :)